docs/Configuration.md

```diff
@@ -907,6 +908,7 @@ config key | description | value type | value default
 `dynamicSearchWindow` | The dynamic search window coefficients used to calculate the EDT(earliest-departure-time), LAT(latest-arrival-time) and SW(raptor-search-window) using heuristics. | object | `null`
 `stopTransferCost` | Use this to set a stop transfer cost for the given [TransferPriority](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.0.0/src/main/java/org/opentripplanner/model/TransferPriority.java). The cost is applied to boarding and alighting at all stops. All stops have a transfer cost priority set, the default is `ALLOWED`. The `stopTransferCost` parameter is optional, but if listed all values must be set. | enum map | `null`
 `transferCacheMaxSize` | The maximum number of distinct transfers parameters (`RoutingRequest`s) to cache pre-calculated transfers for. If too low, requests may be slower. If too high, more memory may be used then required. | int | `25`
+`pagingSearchWindowAdjustments` | The provided array of durations is used to increase the search-window for the next/previous page when the current page return few options. If ZERO results is returned the first duration in the list is used, if ONE result is returned then the second duration is used and so on. The duration is added to the existing search-window and inserted into the next and previous page cursor. See JavaDoc for [TransitTuningParameters#pagingSearchWindowAdjustments](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.0.0/src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/TransitTuningParameters.java) for more info.  | duration[] | `["4h", "1h", "30m"]`
```

Comment:
From looking at the code, isn't the default `["4h", "1h", "30m", "20m", "10m"]`. Also now this doesn't depend on the number of itineraries requested. What happens if this is configured at router-config.json level and someone requests 1 itinerary. Does it increase the search window based on the value configured here for the case where 1 itinerary is returned even though that was the intended result?

Reply:
I think there are 3 cases, of witch I forgot one, in prioritized order where `n` is _number of Itineraries found:
- If the `n` > `numItineraries` requested. The current search-window is cropped and the next/prev window should be cropped as well. 
- The `n`== `numItineraries` requested. We should keep the same window size (i think I missed this case). 
- The `n` < `numItineraries` requested. We should use the `pagingSearchWindowAdjustments` to increase the `searchWindow`.

I will make a fix for this.